dash/dash.py

```diff
@@ -76,9 +76,10 @@ def __init__(
             name='__main__',
             server=None,
             static_folder='static',
-            assets_folder=None,
+            assets_folder='assets',
             assets_url_path='/assets',
             assets_ignore='',
+            assets_blueprint_name='assets',
```

Comment:
How important is this to be a user-configurable param? If we made the initial value `dash-assets` (which would still be prefixed by custom route values -- which was a good move btw) then it seems unlikely that a user would need to modify this.

Just thinking about slowing the rate of kwarg growth for the constructor where possible, as it will get a but unwieldy at some point.

Reply:
Good point let's remove that, there's already much clutter in the kwargs. :ok_hand: